packages-web/badge/src/components/Badge.tsx

```diff
@@ -0,0 +1,35 @@
+import { createElement, ReactElement } from "react";
+import classNames from "classnames";
+
+import "../ui/Badge.css";
+
+export interface BadgeProps {
+    type: "badge" | "label";
+    className?: string;
+    style?: object;
+    value?: string;
+    bootstrapStyle?: BootstrapStyle;
```

Comment:
Use `BrandStyleEnum` provided by the main types (XML)

Reply:
If you do that, you loose the strict separation of display component and Mendix logic.

Reply:
Same values!

Reply:
I understand that the values are the same, but I would like to keep the separation strict, since that's the whole idea of the display component, right?

Reply:
Does it make sense if you have the same values?

Reply:
Already discussed with @Andries-Smit  and we decided to simplify the code because you are just using this component inside Badge. If this one would be re-used in another widget it should be exported inside `packages-common/piw-tools`

Reply:
Changed it to make use of ```BrandStyleEnum```.